typescript/sdk/src/contracts.ts

```diff
@@ -65,55 +72,8 @@ export type RouterAddresses = {
   router: ProxiedAddress;
 };
 
-export const routerFactories = {
+export const routerFactories: Factories<RouterAddresses> = {
```

Comment:
I would prefer if we were not prescriptive about use of `upgradeBeaconController` and the corresponding upgradability pattern.

This setup is useful when you have multiple contracts that share an implementation, that you would like to be able to upgrade atomically, which we do in the `Inboxes`. Abacus applications are less likely to have this property, and are more likely to choose vanilla proxies, or choose to forgo upgradability entirely.

Reply:
https://github.com/abacus-network/abacus-monorepo/issues/373